packages/website/package.json

```diff
@@ -11,6 +11,17 @@
     "Ben Conolly"
   ],
   "license": "MIT",
+  "babel": {
+    "presets": [
+      "@babel/preset-env",
```

Comment:
Having this here makes babel always compile down to es2015 (we don't have any plugin or browserlist configuration to tell it otherwise).

The compiled result references `regeneratorRuntime` which is never loaded, and throws, breaking most of the dev scripts that I've tried.